Assets/HoloToolkit/ARCA/Plugins/WSA/x86/OpenCVWrapper.iobj.meta

```diff
@@ -0,0 +1,9 @@
+fileFormatVersion: 2
```

Comment:
Where does the iobj file come from? Is it from source you wrote? If so, that source needs to be submitted to https://github.com/Microsoft/MixedRealityToolkit. Also, how does this get created?

Reply:
What is `iobj`?